README.md

```diff
@@ -58,7 +58,7 @@ TEST(FirstTestGroup, FirstTest)
 ## Command line switches
 
 * -v verbose, print each test name as it runs
-* -r# repeat the tests some number of times, default is one, default is # is not specified is 2. This is handy if you are experiencing memory leaks related to statics and caches.
+* -r# repeat the tests some number of times, default is one, default if # is not specified is 2. This is handy if you are experiencing memory leaks. A second run that has no leaks indicates that someone
```

Comment:
You're right.  Looks like it had been that way until you fixed it on 2015-05-01 with commit 8414625.  I had originally made my change here before that, so something must've gone wrong when I was rebasing.  Oops- sorry for undoing that.  I'll add the re-fix to my new "minor" branch.